jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -74,6 +94,25 @@ teardown() {
     [ "${EJB_TIMER_XA_CONNECTION_PROPERTY_Url}" = "jdbc:mariadb://myapp-host:3306/rhpam-mariadb?pinGlobalTxToPhysicalConnection=true\&amp;enabledSslProtocolSuites=TLSv1.2" ]
 }
 
+@test "test if the EJB_TIMER datasource has been auto-configured also with RHPAM_XA_CONNECTION_PROPERTY_Url and driver is mariadb" {
+    local expected_timer_service="EJB_TIMER"
+    local expected_datasources="EJB_TIMER,RHPAM"
+    local expected_url="jdbc:mariadb://myapp-host:3306/rhpam-mariadb?enabledSslProtocolSuites=TLSv1.2"
+    export DATASOURCES="RHPAM"
```

Comment:
please use a full datasource configuration, with driver, and the RHPAM XA Url property.

+ please add a behave test.